inst/rmarkdown/templates/lncs/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,107 @@
+# If the paper title is too long for the 
+# running head, you can set an abbreviated 
+# paper title here.
+# titlerunning: "Abbreviated paper title"
+authors: 
+  - name: "First Author"
+    inst: "1"
+    orcid: "0000-1111-2222-3333"
+  - name: "Second Author"
+    inst: "2,3"
+    orcid: "1111-2222-3333-4444"
+  - name: "Third Author"
+    inst: "3"
+    orcid: "2222--3333-4444-5555"
+# First names are abbreviated in the running head.
+# If there are more than two authors, 'et al.' is used.
+authorrunning: "F. Author et al." 
+institutes: 
+  - name: "Princeton University, Princeton NJ 08544, USA"
+  - name: "Springer Heidelberg, Tiergartenstr. 17, 69121 Heidelberg, Germany"
+    email: "lncs@springer.com"
+    url: "http://www.springer.com/gp/computer-science/lncs"
+  - name: "ABC Institute, Rupert-Karls-University Heidelberg, Heidelberg, Germany"
+    email: "\\{abc,lncs\\}@uni-heidelberg.de"
+abstract: | 
+  The abstract should briefly summarize the contents of the paper in 150--250 words.
+keywords: 
+  - First keyword
+  - Second keyword
+  - Another keyword
+bibliography: lncs-test.bib
+
+output: 
+  rticles::lncs_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# First Section
+
+## A Subsection Sample
+
+Please note that the first paragraph of a section or subsection is not indented. 
+The first paragraph that follows a table, figure, equation etc. does not need an indent, either.
+
+Subsequent paragraphs, however, are indented.
+
+### Sample Heading (Third Level)
+
+Only two levels of headings should be numbered. 
+Lower level headings remain unnumbered; they are formatted as run-in headings.
+
+#### Sample Heading (Fourth Level)
+
+The contribution should contain no more than four levels of headings. 
+Table\ \ref{tab1} gives a summary of all heading levels.
+
+\begin{table}
+\caption{Table captions should be placed above the
+tables.}\label{tab1}
+\begin{tabular}{|l|l|l|}
+\hline
+Heading level &  Example & Font size and style\\
+\hline
+Title (centered) &  {\Large\bfseries Lecture Notes} & 14 point, bold\\
+1st-level heading &  {\large\bfseries 1 Introduction} & 12 point, bold\\
+2nd-level heading & {\bfseries 2.1 Printing Area} & 10 point, bold\\
+3rd-level heading & {\bfseries Run-in Heading in Bold.} Text follows & 10 point, bold\\
+4th-level heading & {\itshape Lowest Level Heading.} Text follows & 10 point, italic\\
+\hline
+\end{tabular}
+\end{table}
+
+\noindent Displayed equations are centered and set on a separate line.
+
+\begin{equation}
+x + y = z
+\end{equation}
+
+Please try to avoid rasterized images for line-art diagrams and schemas. 
+Whenever possible, use vector graphics instead (see Fig.\ \ref{fig1}).
+
+\begin{figure}
+\includegraphics[width=\textwidth]{fig1.eps}
+\caption{A figure caption is always placed below the illustration.
+Please note that short captions are centered, while long ones are justified by the macro package automatically.} \label{fig1}
+\end{figure}
```

Comment:
I left it as is because this also shows how to use references, which is, AFAIK, not supported by base rmarkdown.

Reply:
When output is LaTeX only,  referencing should work using `fig.cap` chunk option for figure and for tables, it will also work when `caption` is used in `knitr::kable()`